test/IntegrationTests/SmokeTests.cs

```diff
@@ -295,6 +295,38 @@ public void LogsNoneInstrumentations()
 
         collector.AssertEmpty(5.Seconds());
     }
+
+    [Fact]
+    [Trait("Category", "EndToEnd")]
+    public void TracesNoneInstrumentations()
+    {
+        using var collector = new MockSpansCollector(Output);
+        SetEnvironmentVariable("OTEL_DOTNET_AUTO_TRACES_ENABLED_INSTRUMENTATIONS", "none");
+        RunTestApplication();
+        collector.AssertEmpty(5.Seconds());
+    }
+
+    [Fact]
+    [Trait("Category", "EndToEnd")]
+    public void MetricsNoneInstrumentations()
+    {
+        using var collector = new MockMetricsCollector(Output);
+        SetEnvironmentVariable("OTEL_DOTNET_AUTO_METRICS_ENABLED_INSTRUMENTATIONS", "none");
+        RunTestApplication();
+        collector.AssertEpmty();
+    }
+
+    [Fact]
+    [Trait("Category", "EndToEnd")]
+    public void  AutoLogsDisabledInstrumentationsForILogger()
```

Comment:
There are missing similar tests for `OTEL_DOTNET_AUTO_TRACES_DISABLED_INSTRUMENTATIONS` and `OTEL_DOTNET_AUTO_METRICS_DISABLED_INSTRUMENTATIONS`

Reply:
added 2 more tests... it was bit hard to find how to configure though..